source/unit/Attack.gd

```diff
@@ -11,7 +11,7 @@ export var icon : StreamTexture = null
 onready var specials := get_children()
 
 func execute_before_turn(user, target):
-	if not specials.empty():
+	if not specials == null and not specials.empty():
```

Comment:
`if not special == null and` equals to `if specials and` I think that's much easier to understand.
same goes for lines 24 and 34.

Reply:
fixed, thanks